includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom powered by Wikia",
+				]
+			]
+		],
+		"verticals" => [
+			"links" => [
+				[
+					"type" => "link-branded",
+					"brand" => "tv",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-tv",
+					],
+					"href" => "http://fandom.wikia.com/tv"
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "games",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-games",
+					],
+					"href" => "http://fandom.wikia.com/games",
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "movies",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-movies",
+					],
+					"href" => "http://fandom.wikia.com/movies",
+				],
+			],
+		],
+		"wikis" => [
+			"header" => [
+				"type" => "line-text",
+				"title" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-wikis",
+				],
+			],
+			"links" => [
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-wikis-explore",
+					],
+					"href" => "#",
+				],
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-wikis-communitycentral",
+					],
+					"href" => "#",
+				],
+			],
+		],
+		"search" => [
+			"module" => [
+				"type" => "search",
+				"results" => [
+					"url" => "http://starwars.wikia.com/wiki/Special:Search?fulltext=Search",
+					"param-name" => "query",
+				],
+				"suggestions" => [
+					"url" => "http://starwars.wikia.com/index.php?action=ajax&rs=getLinkSuggest&format=json",
+					"param-name" => "query",
+				],
+				"placeholder-inactive" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-search-placeholder-inactive",
+				],
+				"placeholder-active" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-search-placeholder-active",
+				],
+			],
+		],
+		"anon" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "#",
+				"image" => "wds/anon-avatar",
+				"title" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-userinfo-anon-avatar-title",
+				],
+			],
+			"links" => [
+				[
+					"type" => "authentication-link",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "authentication-link" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)